src/www/head.inc

```diff
@@ -39,6 +39,9 @@ $pagetitle .= html_safe(sprintf(' | %s.%s', $config['system']['hostname'], $conf
     <meta name="description" content="" />
     <meta name="copyright" content="" />
     <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1" />
+    <meta name="mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-touch-fullscreen" content="yes">
```

Comment:
this one seems to work but same as apple-mobile-web-app-capable yet undocumented so probably an artefact from the early days and not required anymore

```suggestion
```